packages/data-models/src/language_model.js

```diff
@@ -369,6 +369,17 @@ class LanguageModel {
     return /^.+\d$/.test(word)
   }
 
+  /**
+   * If last trailing digit is "1" it should be stripped during normalization,
```

Comment:
Can we change this comment to the following:

"Morphological parsers and dictionary indexes may add a trailing digit to disambiguate homonyms. 
These can be ignored for purposes of string comparison."

Reply:
Sure updated that.